Log Viewing/ARKLogTableViewController.m

```diff
@@ -71,6 +75,23 @@ - (instancetype)init;
     return [self initWithLogStore:[ARKLogDistributor defaultDistributor].defaultLogStore logFormatter:[ARKDefaultLogFormatter new]];
 }
 
+- (instancetype)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil;
+{
+    return [self init];
+}
+
+- (instancetype)initWithCoder:(NSCoder *)aDecoder;
+{
+    NSAssert(NO, @"Please use a supported initializer.");
+    return [self init];
+}
+
+- (instancetype)initWithStyle:(UITableViewStyle)style;
+{
+    NSAssert(NO, @"Please use a supported initializer.");
```

Comment:
Marked NS_UNAVAILABLE in the header. On Xcode 6.4 at least, the compiler makes me override too (which makes sense, if a bit pedantic - could call with runtime wizardry)